app/views/admin/tools/index.html.erb

```diff
@@ -4,14 +4,23 @@
   </div>
   <%= link_to(t('tool.new'), new_admin_tool_path, class: 'btn primary') %>
 
-  <div class="list-group">
-    <% @all_tools.each do |tool| %>
-        <a href="<%= edit_admin_tool_path(tool) %>" class="list-group-item">
+  <p class="pull-right"><%= 'Tillgängliga verktyg' %></p>
+  <br>
+
+  <ul class="list-group">
+    <% @all_tools.each do |tool|
+      if tool.free != 0
```

Comment:
This logic would be great to move to an helper, see for example:
`CategoriesHelper`